packages/studio/src/components/PreviewWithUseComponents.tsx

```diff
@@ -4,33 +4,45 @@ import PreviewPanel from "./PreviewPanel";
 import Highlighter from "./Highlighter";
 import IFramePortal from "./IFramePortal";
 import { CSSProperties } from "react";
+import { useState } from "react";
+import { Tooltip } from "react-tooltip";
 
 const inlineStyles: CSSProperties = {
-  transform: "translateY(-50px)",
-};
-
-const innerIframeStyles: CSSProperties = {
-  transform: "translateY(50px)",
+  overflow: "scroll",
 };
 
 export default function PreviewWithUseComponents() {
   const componentTree = useStudioStore((store) =>
     store.actions.getComponentTree()
   );
+  const [tooltipProps, setTooltipProps] = useState({
```

Comment:
can we pass the type of Tooltip's props as the generic type param for the useState call here? that way, you'd be able to simplify the type of setTooltipProps in the other files to just accept any of the Tooltip props